exchanges/graphcache/benchmarks/benchmarks.js

```diff
@@ -0,0 +1,83 @@
+import urqlClient from './urqlClient.js';
+import { ALL_TODOS_QUERY, ALL_WRITERS_QUERY, ALL_BOOKS_QUERY, ALL_STORES_QUERY, ALL_EMPLOYEES_QUERY, ALL_AUTHORS_QUERY, ADD_TODO_MUTATION, UPDATE_TODO_MUTATION, ADD_TODOS_MUTATION, ADD_WRITERS_MUTATION, ADD_BOOKS_MUTATION, ADD_STORES_MUTATION, ADD_EMPLOYEES_MUTATION, ADD_AUTHORS_MUTATION } from "./operations.js";
+
+// create functions that execute operations/queries/mutaitons to be benchmarked
+export const getAllTodos = async () => {
+    const queryResult = await urqlClient.query(ALL_TODOS_QUERY).toPromise();
+    console.log("getAllTodoss Query Result", queryResult);
```

Comment:
Let's remove the logging inside of this file, this changes benchmark results since logging is quite expensive